auth_totp/README.rst

```diff
@@ -0,0 +1,105 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl.html
+   :alt: License: LGPL-3
+
+===========
+MFA Support
```

Comment:
Is it support for MFA (multi-factor authentication) or TOTP? Explain anyway the letters.